armada_command/command_run.py

```diff
@@ -89,17 +89,57 @@ def command_run(args):
     if not microservice_name:
         raise ArmadaCommandException('ERROR: Please specify microservice_name argument'
                                      ' or set MICROSERVICE_NAME environment variable')
+
     ship = args.ship
     is_run_locally = ship is None
     dockyard_alias = args.dockyard or dockyard.get_dockyard_alias(microservice_name, is_run_locally)
+
+    vagrant_dev = _is_vagrant_dev(args.hidden_is_restart, dockyard_alias, microservice_name)
+
+    dockyard_alias, image = _find_dockyard_with_image(vagrant_dev, args.hidden_is_restart, dockyard_alias,
+                                                      microservice_name)
+
+    _print_run_info(image, dockyard_alias, ship, args.rename)
+
+    payload = {'image_path': image.image_path, 'environment': {}, 'ports': {}, 'volumes': {}}
```

Comment:
I would consider creating payload class (maybe even use builder design pattern?). Then the lines below could look like:
```
...
payload.update_hermes(image.image_name, args.env, args.app_id, args.configs)
payload.update_environment(args.e)
...
```